packages/shared_preferences/android/src/main/java/io/flutter/plugins/sharedpreferences/SharedPreferencesPlugin.java

```diff
@@ -116,30 +117,47 @@ private String encodeList(List<String> list) throws IOException {
     return filteredPrefs;
   }
 
+  void backgroundTask(final Editor editor, final MethodChannel.Result result) {
+    new AsyncTask<Void, Void, Boolean>() {
+      @Override
+      protected Boolean doInBackground(Void... voids) {
+        return editor.commit();
+      }
+
+      @Override
+      protected void onPostExecute(Boolean value) {
+        super.onPostExecute(value);
```

Comment:
Is this marked as "must call super"?

Reply:
https://stackoverflow.com/a/20637495/3931968 This says no, so I removed it.